app/forms/catalog_record_id_form.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require "reform/form/coercion"
+
+class CatalogRecordIdForm < Reform::Form
+  ### classes that define a virtual catalog record ID model object and data structure, used in form editing...persistence is in the cocina model
+  class Row
+    attr_accessor :value, :refresh, :catalog, :_destroy
+
+    def initialize(attrs = {})
+      @value = attrs[:value]
+      @refresh = attrs[:refresh]
+      @catalog = attrs[:catalog]
+    end
+
+    def persisted?
+      false
+    end
+  end
+  ###
+
+  feature Reform::Form::Coercion
+
+  collection :catalog_record_ids, populate_if_empty: Row, save: false, virtual: true,
+    prepopulator: ->(*) { catalog_record_ids << CatalogRecordIdForm::Row.new(value: "", refresh: true) if catalog_record_ids.size.zero? } do
+    property :value
+    property :refresh, type: Dry::Types["params.nil"] | Dry::Types["params.bool"]
+    property :_destroy
+  end
+
+  collection :other_catalog_record_ids, save: false, virtual: true do
+    property :value
+    property :catalog
+    property :refresh, type: Dry::Types["params.nil"] | Dry::Types["params.bool"]
+    property :_destroy
+  end
```

Comment:
Added this to help persist Folio catalog links when in Symphony mode, and vice versa.

Reply:
Done. (It won't.)